static/config/99datadog-windows.config

```diff
@@ -0,0 +1,21 @@
+# .ebextensions/99datadog-windows.config
+packages:
+  msi:
+    DotnetAPM: https://github.com/DataDog/dd-trace-dotnet/releases/download/v2.8.0/datadog-dotnet-apm-2.8.0-x64.msi
```

Comment:
Open question to the .NET team is whether there's a link that might auto direct to the latest msi? If not, then I think this one is fine as is.

Reply:
https://github.com/DataDog/dd-trace-dotnet/releases/latest takes you to the latest _release_, but not to the latest _msi_, as we include the version number in the MSI. I don't think there's an easy way to setup a link for that either, but it's worth considering on our end 🤔